src/manage_agent_installers.c

```diff
@@ -403,28 +349,19 @@ static int
 get_agent_installer_data_from_json (cJSON *json,
                                     agent_installer_data_t *data)
 {
-  cJSON *cpes, *cpe_json;
   const char *creation_time_str, *modification_time_str;
 
   memset (data, 0, sizeof(agent_installer_data_t));
 
   GET_AGENT_INSTALLER_JSON_STR (uuid, "uuid");
   GET_AGENT_INSTALLER_JSON_STR (name, "name");
   GET_AGENT_INSTALLER_JSON_STR (description, "description");
-  GET_AGENT_INSTALLER_JSON_STR (content_type, "contentType");
-  GET_AGENT_INSTALLER_JSON_STR (file_extension, "fileExtension");
-  GET_AGENT_INSTALLER_JSON_STR (installer_path, "installerPath");
```

Comment:
I think for consistency with other JSON files in the feed we should keep using camelCase here.